### ISSUE TITLE
resume

### ISSUE THREAD

**Issue by alicera** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#172]
1. If the train break, how to resume it?
I use the command , but it seem to restart train.
> 
Finally, to train from checkpoint from previous step and 3 refinement stages in network, run python train.py --train-images-folder <COCO_HOME>/train2017/ --prepared-train-labels prepared_train_annotation.pkl --val-labels val_subset.json --val-images-folder <COCO_HOME>/val2017/ --checkpoint-path <path_to>/checkpoint_iter_280000.pth --weights-only --num-refinement-stages 3
> 

2.
How to use the num-refinement-stages ?

Assume
First, num-refinement-stages = 1, we took checkpoint after 370000 iterations as the second.

Second, num-refinement-stages = 2, we took checkpoint after 370000 iterations as the third.

Third, num-refinement-stages = 3, we took checkpoint after 370000 iterations as the  final one.

Final, we took num-refinement-stages = 1  checkpoint after final iterations as the final one.

Is it correct?

**Comment by Daniil-Osokin** [comment]
Hi! If you will provide [checkpoint path](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/train.py#L61), resuming will start automatically (in case of you have not specified `--from-mobilenet`)

**Comment by alicera** [comment]
The network resume, but the learning rate?
It start the  iteration 0 ,not the stop iteration.

**Comment by Daniil-Osokin** [comment]
You can check the [code](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/train.py#L68), it resumes properly.

**Comment by alicera** [comment]
I try to train the network with 512x512 and use the checkpoint_370000.pth
I got the error. Other size can work.

[WARNING] Not found pre-trained parameters for model.layer2.2.conv3.weight
[WARNING] Not found pre-trained parameters for model.layer2.2.bn3.weight
[WARNING] Not found pre-trained parameters for model.layer2.2.bn3.bias
[WARNING] Not found pre-trained parameters for model.layer2.2.bn3.running_mean
[WARNING] Not found pre-trained parameters for model.layer2.2.bn3.running_var
[WARNING] Not found pre-trained parameters for model.layer2.2.bn3.num_batches_tracked
[WARNING] Not found pre-trained parameters for model.layer2.3.conv1.weight
[WARNING] Not found pre-trained parameters for model.layer2.3.bn1.weight
[WARNING] Not found pre-trained parameters for model.layer2.3.bn1.bias
[WARNING] Not found pre-trained parameters for model.layer2.3.bn1.running_mean
[WARNING] Not found pre-trained parameters for model.layer2.3.bn1.running_var
[WARNING] Not found pre-trained parameters for model.layer2.3.bn1.num_batches_tracked
[WARNING] Not found pre-trained parameters for model.layer2.3.conv2.weight
[WARNING] Not found pre-trained parameters for model.layer2.3.bn2.weight
[WARNING] Not found pre-trained parameters for model.layer2.3.bn2.bias
[WARNING] Not found pre-trained parameters for model.layer2.3.bn2.running_mean
[WARNING] Not found pre-trained parameters for model.layer2.3.bn2.running_var
[WARNING] Not found pre-trained parameters for model.layer2.3.bn2.num_batches_tracked
[WARNING] Not found pre-trained parameters for model.layer2.3.conv3.weight
[WARNING] Not found pre-trained parameters for model.layer2.3.bn3.weight
[WARNING] Not found pre-trained parameters for model.layer2.3.bn3.bias
[WARNING] Not found pre-trained parameters for model.layer2.3.bn3.running_mean
[WARNING] Not found pre-trained parameters for model.layer2.3.bn3.running_var
[WARNING] Not found pre-trained parameters for model.layer2.3.bn3.num_batches_tracked

**Comment by Daniil-Osokin** [comment]
You train other network, not from this repository. That is why parameters differ.

**Comment by Daniil-Osokin** [comment]
Hope, it helped.
